mne/tests/test_source_estimate.py

```diff
@@ -886,6 +889,39 @@ def test_vec_stc():
     assert_array_equal(normal.data[:, 0], [1, 2, 0, np.sqrt(3)])
 
 
+def test_epochs_vector_inverse():
+    """Test vector inverse consistency between evoked and epochs"""
+
+    raw = read_raw_fif(fname_raw)
+    events = find_events(raw, stim_channel='STI 014')
+    event_id = dict(aud_l=1)  # event trigger and conditions
+    tmin = -0.2  # start of each epoch (200ms before the trigger)
+    tmax = 0.5  # end of each epoch (500ms after the trigger)
+    raw.info['bads'] = ['MEG 2443', 'EEG 053']
+    baseline = (None, 0)  # means from the first instant to t = 0
+    reject = dict(grad=2000e-13, mag=4e-12, eog=150e-6)
+
+    epochs = Epochs(raw, events, event_id, tmin, tmax, baseline=baseline,
+                    reject=reject, preload=True).crop(0, 0.01)
+    evoked = epochs.average(picks=range(len(epochs.ch_names)))
+    # only differs from evoked in STI 001 and STI 014 (not rounded in original)
+
+    inv = read_inverse_operator(fname_inv)
+
+    method = "MNE"
+    snr = 3.
+    lambda2 = 1. / snr ** 2
+
+    stcs_epo = apply_inverse_epochs(epochs, inv, lambda2, method=method,
+                                    pick_ori='vector', return_generator=False)
+    stc_epo = np.mean(stcs_epo)
+
+    stc_evo = apply_inverse(evoked, inv, lambda2, method=method,
+                            pick_ori='vector')
+
+    assert(np.allclose(stc_epo.data,stc_evo.data))
```

Comment:
Just use `assert_allclose` here from `numpy.testing` (it is probably already imported)

Also check to see what values of `rtol` and `atol` can be used -- set as small as possible (to an order of magnitude) that will still pass, and ensure it makes sense relative to the scale of the actual data